src/ring_theory/matrix_equiv_tensor.lean

```diff
@@ -0,0 +1,197 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import ring_theory.tensor_product
+import data.matrix.basic
+
+/-!
+We provide the `R`-algebra structure on `matrix n n A` when `A` is an `R`-algebra,
+and show `matrix n n A ≃ₐ[R] (A ⊗[R] matrix n n R)`.
+-/
+
+universes u v w
+
+open_locale tensor_product
+open_locale big_operators
+
+open tensor_product
+open algebra.tensor_product
+
+variables {R : Type u} [comm_ring R]
+variables {A : Type v} [comm_ring A] [algebra R A]
+variables {n : Type w} [fintype n] [decidable_eq n]
+
+instance : algebra R (matrix n n A) :=
```

Comment:
How about renaming this file to `matrix_algebra.lean`? Then it makes sense that this instance is in the file, and `matrix_equiv_tensor` is also something that makes a lot of sense in a file with the name `matrix_algebra.lean`.